### DISCUSSION TITLE
카카오 웹 로그인 연동 불가 안내

### DISCUSSION THREAD

**Discussion by kimytsc** [kimytsc/covid-rest-vaccine-macro#67] [Announcements]
2021년 8월 19일 00시 기준 카카오 로그인이 막힌 것으로 추정됩니다.

해결책이 강구될때까지 당분간은 네이버를 이용해주시거나 변경된 카카카오 이용방법을 참고해주시길 바랍니다.

**Comment by junehyeok129** [comment]
https://vaccine-map.kakao.com/api/v3/vaccine/left_count_by_coords 여기서 에러가 나는거 같네요ㅜㅠㅠ

**Reply by kimytsc** [reply]
임시방편으로 사용 가능한 방법을 올려두었습니다([사용방법 바로가기](https://github.com/kimytsc/covid-rest-vaccine-macro/tree/main/kakao#mobile)).

**Reply by junehyeok129** [reply]
실행해보았는데 자꾸 네이버 지도창이 나오네요ㅜㅜ

**Reply by kimytsc** [reply]
README의 코드를 그대로 복사해서 테스트해보니 정상적으로 동작하는데, 혹시 코드를 복사한 후 네이버 지도의 URL로 변경하는 과정에서 복사된 URL을 그대로 사용하지 않았을까 생각됩니다.

디버깅을 위해 붙여넣으려고 하셨던 코드 공유 가능할까요?

**Reply by junehyeok129** [reply]
javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?vaccineFilter=unused&x=127.4154995&y=36.3197339&bounds=127.2507046%3B36.2596142%3B127.5802944%3B36.3798071",
    delay: 1000,
    timeout: 3000,
    choice: ["VEN00013","VEN00014"]
}) => {
fetch("https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/kakao/macro.js")
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()
이거입니다!

**Reply by kimytsc** [reply]
전달해주신 코드로 실행해보니 301개 병원 리스트가 노출되며 동작하는걸 확인하였습니다.

혹시 카카오톡 최신 버전으로 업데이트 된 상태이신가요?
